### PR TITLE
feat(pcr0tool): Add verb `pcrread`

### PR DESCRIPTION
**Pull request by xaionaro**
Just a simple tool to read a PCR value from TPM (without using `tpm2-tools`).

Feel free to reject this PR. I only added this PR for a single one-time investigation to differentiate if the bug is in tpm2-tools or with TPM module (or its kernel driver) itself.

### CODE REVIEW

**Comment by ChriMarMe** [comment]
What is the status here @rihter007 @trynity @u1f35c ?

File: cmd/pcr0tool/main.go [hunk 1]
```diff
@@ -33,6 +34,7 @@ var knownCommands = map[string]commands.Command{
 	"dump_registers":               &dumpregisters.Command{},
 	"printnodes":                   &printnodes.Command{},
 	"validate_security":            &validatesecurity.Command{},
+	"pcrread":                      &pcrread.Command{},
```
Comment (trynity) [inline cmd/pcr0tool/main.go:37]: nit: Please sort these commands alphabetically

**Review by trynity — APPROVED** [review]
One minor quibble on the `panic` and a nit, but approving to unblock.

File: cmd/pcr0tool/main.go (hunk 1 above)
Comment (xaionaro) [reply cmd/pcr0tool/main.go]: Fixed.

File: cmd/pcr0tool/commands/pcrread/command.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	assertNoError(err)
+
+	hashAlgo := tpm2.AlgUnknown
+	for _, alg := range []tpm2.Algorithm{tpm2.AlgSHA1, tpm2.AlgSHA256} {
+		if strings.EqualFold(*cmd.hashAlgo, alg.String()) {
+			hashAlgo = alg
+		}
+	}
+	if hashAlgo == tpm2.AlgUnknown {
+		panic(fmt.Errorf("algo '%s' is unknown", *cmd.hashAlgo))
```
[the remark this answers is not in the document]
Comment (xaionaro) [reply cmd/pcr0tool/commands/pcrread/command.go]: If the tool is executed with invalid arguments, then the execution should be interrupted. Replaced with `log.Fatal` to be consistent with the rest of the code. Let me know if you'd like another solution.

**Pull request merged**